mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -2511,6 +2511,41 @@ static bool UpdateAssemblyVersionForAssemblyInfo (XmlElement e, XmlElement root,
 		"System.Runtime.CompilerServices.DynamicAttribute",
 	};
 
+	private IEnumerable<char> FilterSpecialChars (string value)
```

Comment:
Why'd you go for an `IEnumerable<char>` return type and not `string`? Then you could do e.g. `return @"\0";`, and the calling code wouldn't require a `new string (...)` intermediary.